packages/components/pie-button/src/button.scss

```diff
@@ -241,6 +268,9 @@ $breakpoint-wide: 768px;
         }
     }
 
+    // ************
+    // Focus styles
+    // ************
```

Comment:
I am sure the answer is yes, but does vite strip these comments out?

Reply:
Yeh, they are Sass comments, so not valid CSS, so get stripped out in the compile step. Only `/* */` style comments would be left in.